examples/quick-start-simple/atmos.yaml

```diff
@@ -17,5 +17,4 @@ stacks:
   name_pattern: "{stage}"
 
 logs:
-  file: "/dev/stderr"
```

Comment:
I suggest removing this from quick start simple 

Especially because this config causes all the logs to direct towards error and a quick start guide should not do that.

Of course we can document and give a seperate example (which i found many already in the repo) for this behavior.

Reply:
Based on the code in the following two places;
https://github.com/cloudposse/atmos/blob/main/pkg/logger/logger.go#L61-L98
https://github.com/cloudposse/atmos/blob/main/pkg/utils/log_utils.go#L106-L143

The log emitted is not based on the log level if we set the log file to stderr or stdout.
This caused my code with `u.LogInfo` to be written in stderr terminal.

I assumed this is some required behaviour given that the file path were prefixed with `/dev`
although in hindsight it seems why we would require this code at all?

Log level is all we should be needing.

Just FYI, I think due to above behavior we are using empty atmosConfig here; https://github.com/cloudposse/atmos/blob/main/cmd/cmd_utils.go#L170-L199

Though this would work in cases where user did not set a file for log, it won't if he sets a file for log because using empty config should not be great.